src/constants.js

```diff
@@ -76,3 +76,7 @@ export const REGEX_RULES = {
   specialCharsRule: /^[a-zA-Z0-9_\-.'*~\s]+$/,
   noSpaceRule: /^\S*$/,
 };
+
+export const IFRAME_FEATURE_POLICY = (
+  'microphone *; camera *; midi *; geolocation *; encrypted-media *, clipboard-write *'
```

Comment:
I mentioned this on another PR, but this contains a typo: https://github.com/openedx/frontend-app-learning/pull/1500

Reply:
[inform] Implemented in follow-up commit: https://github.com/openedx/frontend-app-authoring/pull/1431/commits/ea4da9f7625c3ca176d5fd8132f0b23a8329886c#diff-8e3b187293d8e7902d08eff3c650466860fe527b0c7da8ec32c1ee4a450f97c5R81